app/code/community/Bolt/Boltpay/Helper/GraphQL/Constants.php

```diff
@@ -0,0 +1,27 @@
+<?php
+class Bolt_Boltpay_Helper_GraphQL_Constants {
```

Comment:
Since we are only defining constansts, make this an `interface` instead of a `class`.

Ideally, this would be located under the `lib/Boltpay/GraphQL` directory.

Reply:
done